serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -154,13 +165,16 @@ Alternatively, you can only pass the necessary data:
 }
 ----
 
-The `JsonNode` that the function returns will be the new payload from this point of the workflow execution.
+In the previous example, the `JsonNode` that the `java` function returns is the new payload in the Serverless Workflow execution.
 
-WARNING: Refrain from using **only** Java functions or use them to call external services. Try to use the xref:service-orchestration/orchestration-of-opnapi-based-services.adoc[the services orchestration features instead].
+[WARNING]
+====
+Avoid using only `java` functions or you can use the `java` functions to call the external services. Also, you can use the xref:service-orchestration/orchestration-of-opnapi-based-services.adoc[services orchestration features].
```

Comment:
@hmanwani-rh I think this change lost the meaning of the warning. What we mean here is that users must avoid using `java` functions to call external services. They should use orchestration instead.

Reply:
@ricardozanini oh, thanks for letting me know. 
I've changed the warning as follows:

`Avoid using `java` functions to call the external services, however, you can use the xref:service-orchestration/orchestration-of-opnapi-based-services.adoc[services orchestration features].`